planning/behavior_path_planner/src/scene_module/lane_change/avoidance_by_lane_change.cpp

```diff
@@ -179,7 +186,7 @@ ObjectData AvoidanceByLaneChange::createObjectData(
   const auto object_closest_index = findNearestIndex(path_points, object_pose.position);
   const auto object_closest_pose = path_points.at(object_closest_index).point.pose;
   const auto t = utils::getHighestProbLabel(object.classification);
-  const auto object_parameter = p->object_parameters.at(t);
+  const auto object_parameter = avoidance_parameters_->object_parameters.at(t);
```

Comment:
```suggestion
  const auto & object_parameter = avoidance_parameters_->object_parameters.at(t);
```

Suggest adding reference.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/4258/commits/2e2c965a02c4a61cd80818e1565be6f0a707c88c.